__tests__/src/lib/AnnotationItem.test.js

```diff
@@ -124,6 +130,10 @@ describe('AnnotationItem', () => {
       expect(new AnnotationItem({ target: 'www.example.com/#xywh=10,10,100,200' })
         .fragmentSelector).toEqual([10, 10, 100, 200]);
     });
+    it('multiple selectors', () => {
+      expect(new AnnotationItem({ target: { selector: [{ type: 'FragmentSelector', value: 'www.example.com/#xywh=10,10,100,200' }] } })
```

Comment:
AFAIK the `value` of a `FragmentSelector` is just the fragment, i.e. `value: 'xywh=10,10,100,200'` ;-)

Reply:
Thanks for that @robcast. I've updated the test to reflect that.